cargo-apk/README.md

```diff
@@ -73,6 +73,25 @@ version = 1
 name = "android.permission.WRITE_EXTERNAL_STORAGE"
 max_sdk_version = 18
 
+# See https://developer.android.com/guide/topics/manifest/uses-permission-element
+#
+# Note: there can be multiple .uses_permission entries.
+[[package.metadata.android.uses_permission]]
+name = "org.khronos.openxr.permission.OPENXR"
+
+# See https://developer.android.com/guide/topics/manifest/uses-permission-element
+#
+# Note: there can be multiple .uses_permission entries.
+[[package.metadata.android.uses_permission]]
+name = "org.khronos.openxr.permission.OPENXR_SYSTEM"
```

Comment:
The `.uses_permission` has already been documented above, no need to document it twice more?

Reply:
Yes I put it in there anyway because the exammple queries relates to those permssions. I put them in for consistency but I can take them out if you reall want.

Reply:
Removed.